Frameworks/Foundation/NSIndexSet.mm

```diff
@@ -64,7 +68,7 @@ - (instancetype)initWithIndex:(NSUInteger)index {
 */
 - (instancetype)initWithIndexesInRange:(NSRange)indexRange {
     if (self = [super init]) {
-        [(NSMutableIndexSet*)self _addItem:indexRange];
+        [static_cast<NSMutableIndexSet*>(self) _addItem:indexRange];
```

Comment:
wrt changing casts? I ended up just changing them all when I fixed mine

---
In reply to: [81208628](https://github.com/Microsoft/WinObjC/pull/1051#discussion_r81208628) [](ancestors = 81208628)